packages/browser-wallet/src/popup/page-layouts/FullscreenPromptLayout/FullscreenPromptLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link, Outlet } from 'react-router-dom';
+import { absoluteRoutes } from '@popup/constants/routes';
+
+export default function MainLayout() {
```

Comment:
```suggestion
export default function FullscreenPromptLayout() {
```

Reply:
Also what is the distinction between pages and page-layouts? is this not this not just also a page?